schemas/fsssignaldiscovered-v1.0.json

```diff
@@ -0,0 +1,81 @@
+        },
+        "header": {
+            "type"                  : "object",
+            "additionalProperties"  : true,
+            "required"              : [ "uploaderID", "softwareName", "softwareVersion" ],
+            "properties"            : {
+                "uploaderID": {
+                    "type"          : "string"
+                },
+                "softwareName": {
+                    "type"          : "string"
+                },
+                "softwareVersion": {
+                    "type"          : "string"
+                },
+                "gatewayTimestamp": {
+                    "type"          : "string",
+                    "format"        : "date-time",
+                    "description"   : "Timestamp upon receipt at the gateway. If present, this property will be overwritten by the gateway; submitters are not intended to populate this property."
+                }
+            }
+        },
+        "message": {
+            "type"                  : "object",
+            "description"           : "Contains all properties from the listed events in the client's journal minus Localised strings and the properties marked below as 'disallowed'",
+            "additionalProperties"  : false,
+            "required"              : [ "event", "signals"],
+            "properties"            : {
+                "event": {
+                    "enum"          : [ "FSSSignalDiscovered" ]
+                },
+                "horizons": {
+                    "type"          : "boolean",
+                    "description"   : "Whether the sending Cmdr has a Horizons pass."
+                },
+                "odyssey": {
+                    "type"      : "boolean",
+                    "description" : "Whether the sending Cmdr has an Odyssey expansion."
+                },
+                "signals": {
+                    "type": "array",
+                    "description": "Array of FSSSignalDiscovered events",
+                    "minItems": 1,
+                    "items": {
+                        "type": "object",
+                        "required": ["SystemAddress", "timestamp", "SignalName"],
+                        "description": "Single FSSSignalDiscovered event",
+                        "properties": {
+                            "SystemAddress": { "type": "integer" },
+                            "timestamp": {
+                                "type"          : "string",
+                                "format"        : "date-time"
+                            },
+                            "SignalName": { "type":  "string" },
+                            "IsStation": { "type": "boolean" },
+                            "TimeRemaining": {"$ref" : "#/definitions/disallowed"},
+
+                            "patternProperties": {
+                                "_Localised$"       : { "$ref" : "#/definitions/disallowed" }
+                        }
+                    }

```

Comment:
Needs to allow optional StarPos and SystemName, for those senders who want to do the matching up of SystemAddress to send them.  But it's only 'SHOULD', so not mandatory and listeners should cope with only getting SystemAddress.